Examples/Scripts/Python/full_chain_itk.py

```diff
@@ -10,11 +10,20 @@
 field = acts.examples.MagneticFieldMapXyz(str(geo_dir / "bfield/ATLAS-BField-xyz.root"))
```

Comment:
I'm actually commenting about the previous line (line 9), but I can't get GitHub to select it.
This should be:
```
detector, trackingGeometry, decorators = acts.examples.itk.buildITkGeometry(geo_dir)
```
and the import should replace `itk` with `acts.examples.itk`.

Reply:
and line 51, the commented-out import:
```
# from acts.examples.reconstruction import SeedingAlgorithm, ParticleSmearingSigmas
```
(this really is quite annoying that GitHub doesn't let you make suggestions for additional changes)

Reply:
addressing in the next commit

Reply:
Fixed